build.gradle

```diff
@@ -82,7 +95,7 @@ android {
         nightly {
             signingConfig signingConfigs.nightly
             applicationIdSuffix ".nightly"
-            versionNameSuffix "-nightly"
+            versionNameSuffix "-nightly-" + GitCommit.currentShort()
```

Comment:
is overriden again later?

Reply:
That can be removed indeed. Leftover from my previous attempts.